Makefile

```diff
@@ -149,16 +153,39 @@ help:
 	@echo " kvasir                   -- make Kvasir, the C front end"
 	@echo " very-clean               -- remove (most) all generated files"
 	@echo " dyncomp-jdk              -- Make file java/dcomp_rt.jar"
-	@echo "Creating the Daikon distribution:"
+	@echo " quick-test               -- simple verification of system"
+	@echo
+	@echo "Targets for creating the Daikon distribution:"
 	@echo " daikon.tar daikon.jar    -- just makes the tar files"
 	@echo " staging                  -- moves all release file to staging-daikon/"
 	@echo " test-staged-dist         -- tests the distribution in staging-daikon/"
 	@echo " staging-to-www           -- copies staging-daikon/ to website"
 	@echo " "
-	@echo "This Makefile is for manipulating the entire Daikon system."
 	@echo "Daikon source code is in the java/daikon subdirectory (but you"
 	@echo "can perform basic operations like compiling it from here)."
 
+distribution-check:
```

Comment:
Shouldn't this be located in the file near the distribution targets?  And quick-test near the test targets?